### DISCUSSION TITLE
Welcome to skylon Discussions!

### DISCUSSION THREAD

**Discussion by sujaldev** [sujaldev/skylon#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by aarudhrakkaran** [comment]
Do you have any examples for a similar project for  atm

**Reply by sujaldev** [reply]
You want to run a browser inside an ATM machine?

**Reply by aarudhrakkaran** [reply]
I would like some assistance on creating a atm machine program that contains elements like name, balance, withdrawal, customer-id ,etc..,
it should be done by combining MySQL and python
.
